src/components/CLI/CLI.tsx

```diff
@@ -42,6 +61,33 @@ export default function Cli({ decreaseCommandsLeft }: CliProps) {
   const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Should this name be updated?

Reply:
In my opinion the name justifies the action, as we are handling `keyDown` events, Moreover we don't have any other function conflicting similar to this name

Even typing commands can be considered as `keyDown` event, though we're handling it directly with the dispatch action